### PR TITLE
Reuse viewer rebased

### PR DESCRIPTION
**Pull request by 3top1a**
Rebased https://github.com/PrismarineJS/prismarine-viewer/pull/139

Tested, it works without any modifications.

### CODE REVIEW

**Comment by rom1504** [comment]
can you adapt the doc ?

**Comment by 3top1a** [comment]
The use seems to be the same, I tested it with my personal project, and it worked.

**Comment by rom1504** [comment]
it returns initialize now
that's the change that needs to be documented

**Comment by 3top1a** [comment]
Where can I document it?

File: lib/mineflayer.js [hunk 1]
```diff
@@ -14,78 +17,74 @@ module.exports = (bot, { viewDistance = 6, firstPerson = false, port = 3000, pre
...
     })
-  })
+  }
+  initialize(bot)
 
   http.listen(port, () => {
     console.log(`Prismarine viewer web server running on *:${port}`)
   })
 
-  bot.viewer.close = () => {
```
Comment (rom1504) [inline lib/mineflayer.js:85]: why did you remove this ?

Comment (3top1a) [reply lib/mineflayer.js:85]: Sorry for not responding for a while, as I said, this is just a rebase of that pull.
I'll add it back in

**Comment by 3top1a** [comment]
I give up

**Comment by rom1504** [comment]
use npm run fix

**Comment by 3top1a** [comment]
Either I don't know npm, or it doesn't know tf is going on either considering it didn't change anything.

**Comment by 3top1a** [comment]
When I run it on my test script, it shows the entities but not the world

**Comment by 3top1a** [comment]
When I run it on my test script, it shows the entities but not the world

**Comment by 3top1a** [comment]
https://user-images.githubusercontent.com/57371001/111629930-b6dd0580-87e9-11eb-8342-9cdff6ec3c91.mp4

**Comment by rom1504** [comment]
stale
